sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -276,12 +278,21 @@ export class KeysClient {
       delete unflattenedOptions.expires;
       delete unflattenedOptions.requestOptions;
 
+      const spanOptions = this.getSpanOptions(unflattenedOptions);
```

Comment:
This won't get the span options because `requestOptions` has been deleted from the `unflattenedOptions` in the line before this line